node/pkg/tss/broadcast_logic.go

```diff
@@ -0,0 +1,131 @@
+package tss
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	tsscommv1 "github.com/certusone/wormhole/node/pkg/proto/tsscomm/v1"
+	"github.com/yossigi/tss-lib/v2/tss"
+)
+
+// The following code follows Bracha's reliable broadcast algorithm.
+
+// voterId is comprised from the id and key of the signer, should match the guardians (in GuardianStorage) id and key.
+type voterId struct {
+	id  string
+	key string
```

Comment:
Yes, `key=tss.PartyId.Key` which should be the PK.
I'll use the ID only since it is the only field that must be unique.

The code inspects for both in more places to ensure `partyId` equality. Should I remove the check for PK from there, too?

Reply:
I think it's okay to check the key too. Maybe just rename key to pubkey